bootstrap-devbox.sh

```diff
@@ -37,44 +46,37 @@ if ! ssh-add -l >/dev/null 2>&1; then
   exit 1
 fi
 
-cd "$HOME/inca"
-echomsg '#################################'
-echomsg 'Updating Inca git repo'
-echomsg '#################################'
-git checkout dev
-if ! git pull origin dev; then
-  echo
-  echoerr "Could not update Inca git repo!"
-  echo
-  echowarn "Do you have your Github SSH key loaded added to your ssh agent keychain?"
-  echowarn "Run ${GRN}ssh-add -l${YLW} to check if your Github key is loaded."
-  echo
-  exit 1
-fi
+update_git_repo() {
+  declare which_repo="$1"
+  echomsg '#################################'
```

Comment:
shouldnt the `cd ${HOME}/$1` be here? that would make this method more self contained

Reply:
friendly name is different than directory name; didn't want to change output to save 1 line